src/Application/Notifier/StripeChatterInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace MatchBot\Application\Notifier;
+
+use Symfony\Component\Notifier\ChatterInterface;
+
+interface StripeChatterInterface extends ChatterInterface
+{}
```

Comment:
Feels strange to extend the interface without adding anything. This is just so that the container can distinguish the two instances and know where to inject each one right? Not a blocker but I feel like there should be a nicer way to do that.

And actually if we are gong the way of extending it don't we really need to give it two children - one `StripeChatterInterface` and one `MatchbotChatterInterface`? Otherwise it looks like anything that declares a dependency on the parent interface doesn't care which channel its messages go to.

Reply:
Yes, that's why. I think it's OK for consumers to essentially "not care" for now, it's not the end of the world if we have a default channel that doesn't require much thought. I'd be interested in other options than extending the interface but couldn't immediately think of one that would work well with Slim DI. This felt like an improvement on essentially doing DI stuff inside the payout handler as we were before.

Reply:
It's not that we shouldn't have a default channel, it's that the the consumers don't specify that they want the default channel. If the parent interface promises to give them a default channel then to be a Liskov-subtype the child interface should give them the default channel too. Can leave it for now, although I think making the third interface would make it easier to understand.